routes/drones.js

```diff
@@ -1,36 +1,50 @@
 const express = require('express');
```

Comment:
Hola miguel me di cuenta que tienes muchas rutas con cntrollers que todavia no estan definidos, ten cuidado con esto empieza con una funcionalidad y luego te pasas a otra 👀 
```suggestion
const express = require("express");
const router = express.Router();

const droneController = require("./../controllers/droneController");

// require the Drone model here

router.get("/drones", droneController.getAllDrones);
router.get("/drones/create", droneController.viewCreateDrone);
router.post("/drones/create", droneController.createDrone);

module.exports = router;
```